package.json

```diff
@@ -27,13 +27,15 @@
   "scripts": {
     "format": "prettier --write 'packages/**/*.{ts,tsx,js,jsx,json,md}'",
     "bootstrap": "lerna bootstrap --hoist",
-    "lint": "tslint --project .",
+    "lint": "lerna run lint",
     "test": "lerna run test",
     "build": "lerna run build",
     "prepublish": "npm run build",
-    "publish": "lerna publish"
+    "publish": "lerna publish",
+    "lint:fix": "eslint 'packages/**/*.{ts,tsx}'"
```

Comment:
Doesn't seem like it's fixing anything. 🤔

Reply:
Yup, change it here 60b7d0a9a92fa8a9a32b634abcf23f731877a3da